packages/console/src/components/SubmitFormChangesActionBar/index.tsx

```diff
@@ -1,17 +1,30 @@
 import Button from '../Button';
+import type { Props as ButtonProps } from '../Button';
 import * as styles from './index.module.scss';
 
 type Props = {
   isOpen: boolean;
   isSubmitting: boolean;
+  onSubmit?: () => Promise<void>;
   onDiscard: () => void;
 };
 
-const SubmitFormChangesActionBar = ({ isOpen, isSubmitting, onDiscard }: Props) => {
+const SubmitFormChangesActionBar = ({ isOpen, isSubmitting, onSubmit, onDiscard }: Props) => {
   if (!isOpen) {
     return null;
   }
 
+  const submitButtonProps: Partial<ButtonProps> = onSubmit
+    ? {
+        htmlType: 'button',
+        onClick: async () => {
+          await onSubmit();
+        },
+      }
+    : {
+        htmlType: 'submit',
+      };
+
```

Comment:
is it possible to customize the form submit method directly instead of tweaking the submit button logic here?

Reply:
Refactored.
Use a `form id` to relate this `submit` button to a form.